.gitmodules

```diff
@@ -324,3 +324,7 @@
 	path = third_party/infineon/psoc6/psoc6_sdk/libs/lwip-network-interface-integration
 	url = https://github.com/Infineon/lwip-network-interface-integration.git
 	platforms = infineon
+[submodule "third_party/lwip/repo"]
+	path = third_party/lwip/repo
+	url = https://github.com/lwip-tcpip/lwip.git
+	branch = master
```

Comment:
after bouffalolab change I think only ti targets are using it. added commit on top with this change

Reply:
after checking a bit- lwip is on by default for many builds, it might be a good idea to clean it up but as im leaving samsung by EOM i wont have the bandwidth to do so.  Reverted the platforms= addition in gitmodules. I think we might speed bootstrap up (at least for CI by cloning submodules with either --filter=tree:0 or trying shallow clones with --depth=1 but the latter might bring [some issues](https://git-scm.com/docs/shallow)